sr/comp/mixtape/mixtape.py

```diff
@@ -17,6 +17,13 @@ def preload(filename: str):
         f.read(1)
 
 
+def populate_filename_placeholder(filename: str, match_num: int) -> str:
+    try:
+        return filename.format(match_num=match_num)
+    except KeyError:
```

Comment:
What's the purpose of this check here?

Reply:
This just means we don't mangle filenames that contain curly braces but aren't actually placeholders.

Reply:
Removed in f399e5560c155958f944a37bb5565134355dbb35